haystack/nodes/answer_generator/openai.py

```diff
@@ -1,15 +1,27 @@
-from typing import Optional, List, Tuple, Union
+# pylint: disable=missing-timeout
```

Comment:
This was recently removed from this PR https://github.com/deepset-ai/haystack/pull/3895/files#diff-c1dbd595081731082bbbe0e65673b6d4831b0da5547babd7a95ecd9c85649c3d could we re-remove this pylint disable?